test/TestImportWithMalformed.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+import os
+
+import pytest
+
+from ansiblelint import Runner, RulesCollection
+
+
+PlayFile = namedtuple('PlayFile', ['name', 'content'])
+
+
+IMPORT_TASKS_MAIN = PlayFile('import-tasks-main.yml', '''
+- oops this is invalid
```

Comment:
I think I'd add another playbook but with `- ping:` instead and add that to parametrize